lib/miq_automation_engine/service_models/miq_ae_service_vm_infra.rb

```diff
@@ -1,4 +1,6 @@
 module MiqAeMethodService
   class MiqAeServiceVmInfra < MiqAeServiceVm
+    expose :set_remote_console_url
```

Comment:
Based on the active record model method this belongs in miq_ae_service_vm.rb.  (Should be removed from `miq_ae_service_vm_infra.rb` and `miq_ae_service_vm_or_template.rb`.